src/components/BackgroundDiv.tsx

```diff
@@ -16,10 +19,18 @@ const StyledBackgroundDiv = styled(({ backgroundColor, ...rest }) => <div {...re
   ${({ backgroundColor }) => `background-color: ${backgroundColor}`}
 `;
 
-const BackgroundDiv = ({ children, ...props }: BakcgroundDivProps) => (
-  <StyledBackgroundDiv {...props}>
-    <ContrastContent backgroundColor={props.backgroundColor}>{children}</ContrastContent>
-  </StyledBackgroundDiv>
-);
+const BackgroundDiv = ({ children, ...props }: BakcgroundDivProps) => {
+  const darkMode = isBackgroundColorDark(props.backgroundColor);
+
+  return (
+    <StyledBackgroundDiv {...props}>
+      {darkMode ? (
+        <MuiThemeProvider theme={contrastTheme}>{children}</MuiThemeProvider>
+      ) : (
+        <MuiThemeProvider theme={theme}>{children}</MuiThemeProvider>
```

Comment:
blir dette det samme? theme er jo allerede satt lenger opp i treet
```suggestion
        children
```

Reply:
åja, så dette trengs feks når vi legger til forfatter da, siden vi der har light -> dark -> (trenger light igjen)?

Reply:
yes 👌